config/locales/en.yml

```diff
@@ -171,13 +171,7 @@ en:
         moj: Select all that apply
         form_moj_other_date_from: For example, 3 11 2000
         form_moj_other_date_to: For example, 12 5 2014
-        letter_of_consent: |
-          You must provide a letter of consent from the subject of this request which:
-            • states that they consent to their information being released to you
-            • is signed by them
-            • is dated within the last 6 months
-          There is no official template
-        letter_of_consent_file_size: "Maximum size: 7 MB"
+        letter_of_consent_file_size: "Maximum size: 7 MB."
```

Comment:
For consistency the file size needs to have a space i.e. `7 MB`

Reply:
it does doesn't it?

Reply:
Yes, I guess it does! Not sure why I thought otherwise.

Reply:
Although this does need to be defined as the hint.